packages/ui/src/common/components/Modal/Modal.tsx

```diff
@@ -179,10 +180,25 @@ export const ModalBody = styled.div`
   border-bottom: 1px solid ${Colors.Black[200]};
 `
 
-export const ModalFooter = styled.footer<{ twoColumns?: boolean }>`
+interface ModalFooterProps {
+  twoColumns?: boolean
+  children?: ReactNode
+  className?: string
+}
+
+export const ModalFooter = ({ twoColumns = false, children, className }: ModalFooterProps) => {
+  return (
+    <ModalFooterComponent twoColumns={twoColumns} className={className}>
+      {children}
+      <ConnectionStatusDot onlyPerformance />
+    </ModalFooterComponent>
+  )
+}
+
+export const ModalFooterComponent = styled.footer<{ twoColumns?: boolean }>`
   display: inline-grid;
   grid-area: modalfooter;
-  grid-template-columns: ${({ twoColumns }) => (twoColumns ? '1fr auto' : '1fr')};
+  grid-template-columns: ${({ twoColumns }) => (twoColumns ? '1fr auto auto' : '1fr auto')};
```

Comment:
This looks weird when there is no connection issues:
![image](https://user-images.githubusercontent.com/6571453/190390844-e38c62a8-a8c6-4552-b7e0-0aac66bbe756.png)


IMO it would be nicer to have the `ConnectionStatusDot` push the button to the left only when it appears.